example/create_dependencies.py

```diff
@@ -60,10 +60,23 @@ def usage():
     """This function prints the usage information and exits. It is called if
     incorrect input parameters are supplied.
     """
-    print("{0} file1 [file2...]".format(sys.argv[0]))
+    print("{argv[0]} file1 [file2...]")
     sys.exit(-1)
 
 
+# -----------------------------------------------------------------------------
+def get_root(file_name):
+    """This function returns the 'root' of a file, which is the filename
+    without any path component and without an extension. E.g.:
+    `dir1/dir2/myfile.x90` would return `myfile`.
+    :param str file_name: the name of the file.
+    :returns: the root of the name of the file.
```

Comment:
Blank line before :returns: and :param: please.

Reply:
Oops (I am pretty certain I tried to look this up, and when I didn't find a coding style, I looked at some random files ... which I guess where outdated). I also moved the empty line into the comment (I so wish `black` would complain about this :) ).